homeassistant/components/modbus/base_platform.py

```diff
@@ -259,9 +325,14 @@ async def async_added_to_hass(self) -> None:
 
     async def async_turn(self, command: int) -> None:
         """Evaluate switch result."""
+        if self._call_active:
+            return
```

Comment:
this block is not needed that is done at a lower level.

Reply:
Remark my review is not complete, just to give you some items.

Reply:
Thanks.
I did a rebase before opening the PR